e2e/container_registry_test.go

```diff
@@ -110,6 +110,34 @@ var _ = Describe("enforcing a Container Registry", func() {
 		}).Should(Succeed())
 	})
 
+	It("should deny patching a not matching registry after applying with a matching", func() {
+		ns := NewNamespace("registry-patch")
+		NamespaceCreation(ns, tnt.Spec.Owners[0], defaultTimeoutInterval).Should(Succeed())
+
+		pod := &corev1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "container",
+			},
+			Spec: corev1.PodSpec{
+				Containers: []corev1.Container{
+					{
+						Name:  "container",
+						Image: "myregistry.azurecr.io/myapp:latest",
+					},
+				},
+			},
+		}
+
+		cs := ownerClient(tnt.Spec.Owners[0])
+		_, err := cs.CoreV1().Pods(ns.Name).Create(context.Background(), pod, metav1.CreateOptions{})
+		if err == nil {
+			c := pod.DeepCopy()
+			c.Spec.Containers[0].Image = "attacker/google-containers/pause-amd64:3.0"
+			_, err := cs.CoreV1().Pods(ns.Name).Update(context.Background(), c, metav1.UpdateOptions{})
```

Comment:
@prometherion I need your help here, i don't get it running. I just want to simulate, that a capsule user patches the image of a pod with an attacker registry. But it doesn't work.

Reply:
I'll try to debug your upstream ASAP and get back to you! 👍🏻

Reply:
May I ask you to rebase over the latest changes on `master` and ensure first the manifests are aligned?